force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -31,9 +32,11 @@ public with sharing class VariablesDataTypesOperators {
      * @param b The second number.
      * @return The sum of the two numbers, or null if either number is null.
      */
-    public static Integer addition(Integer a, Integer b) {  
+    public static Integer addition(Integer a, Integer b) { 
+        if (a== null || b == null) {
```

Comment:
Great work. Try to be consistent with spacing around your operators just for easier readability. 
`if (a == null || b == null) {`